armi/bookkeeping/db/tests/test_comparedb3.py

```diff
@@ -155,7 +155,7 @@ def test_compareDatabaseSim(self):
 
         # end-to-end validation that comparing a photocopy database works
         diffs = compareDatabases(dbs[0]._fullPath, dbs[1]._fullPath)
-        self.assertEqual(len(diffs.diffs), 459)
+        self.assertEqual(len(diffs.diffs), 456)
```

Comment:
I feel like I was a bit too glib when I wrote this unit test.

I should probably re-write it so the actual `assert`s are a little more meaningful and transparent.